src/gui_common/charts/line/LineChart.cs

```diff
@@ -299,11 +319,13 @@ public void ClearDataSets()
                 visibleDataSetCount++;
             }
 
+            // Initialize line
+            var dataLine = new DataLine(data.Value, data.Key == defaultDataSet);
+            dataLines[data.Key] = dataLine;
```

Comment:
Not sure why resharper is complaining here, seems pretty normal to me...

Reply:
What if you combine the above initialization and this line? might also be related to data.Key. Looking at the XML report this is the reported character range in the file that is problematic: `9832-9840`.

Reply:
Using emacs, I found that the character range corresponds to `[data.Key]` so resharper thinks that "data.Key" might be null here.

Reply:
Hm, that's weird considering dictionary keys can't be null. I guess will add a little null check and see if it'll work.

Reply:
Maybe the fact that data.Key comes from DataSetDictionary confuses resharper enough to not know that it isn't null. Btw rider locally also shows me the same thing so this isn't caused by the CI running quite an outdated version of the resharper tools.

Reply:
The null check fixed it but yeah that's interesting.